docs/source/format_release_notes.rst

```diff
@@ -1,6 +1,11 @@
 hdmf-common Release Notes
 =========================
 
+1.2.2 (Upcoming)
```

Comment:
Is an update to documentation something that warrants a version bump?

Reply:
No. Good point. I was treating this as a repo changelog but this is not that. I reverted the version bump.